File: RecycleView22/app/src/main/java/com/example/recycleview2/MainActivity.java
package com.example.recycleview2;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<ModalClass> List = new ArrayList<>();

    ArrayList<String> usernameList = new ArrayList<>();
    ArrayList<Integer> usernumberList = new ArrayList<>();

    RecyclerView rcvUser;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initview();
    }

    private void initview()
    {

        setdetailList();

        for (int i = 0; i <usernameList.size(); i++)
        {
            ModalClass model = new ModalClass();
            model.setUsername(usernameList.get(i));
            model.setNumber(usernumberList.get(i));
            List.add(model);
        }

        rcvUser = findViewById(R.id.rcvUser);

        UserAdapter adapter = new UserAdapter(List);

        LinearLayoutManager manager = new LinearLayoutManager(this,LinearLayoutManager.VERTICAL,false);
        rcvUser.setLayoutManager(manager);
        rcvUser.setAdapter(adapter);

    }


    private void setdetailList() {
        usernameList.add("Sanjay");
        usernameList.add("Himanshu");
        usernameList.add("Purvish");
        usernameList.add("Dikshit");
        usernameList.add("Shruti");
        usernameList.add("Dhruv");
        usernameList.add("Vensi");
        usernameList.add("Akshay");
        usernameList.add("Maulik");
        usernameList.add("Hasu");
        usernameList.add("Jenish");
        usernameList.add("Ronak");
        usernameList.add("Pratik");
        usernameList.add("Akshar");
        usernameList.add("Vinit");

        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
        usernumberList.add(901628593);
    }
}

File: Project2/app/src/main/java/com/example/project_2/MainActivity.kt
package com.example.project_2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

//    var name : String = "Dikshit"
//    var id : Int = 1
//    var english : Int = 78
//    var maths : Int = 60
//    var hindi : Int = 75
//    var gujarati : Int = 80
//    var Tecnology : Int = 67
//    var mydata : Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initview()

    }

    private fun initview() {

       var standard1 = ModalClass(1,"Akshay")
       var standard2 = ModalClass(1,"Brijesh")
       var standard3 = ModalClass(1,"Chandan")
       var standard4 = ModalClass(1,"Dikshit")
       var standard5 = ModalClass(1,"Elvish")

        Log.e("TAG", "initview: "+standard1.id+"="+standard1.name)
        Log.e("TAG", "initview: "+standard2.id+"="+standard2.name)
        Log.e("TAG", "initview: "+standard3.id+"="+standard3.name)
        Log.e("TAG", "initview: "+standard4.id+"="+standard4.name)
        Log.e("TAG", "initview: "+standard5.id+"="+standard5.name)

    }
}